scripts/deploy-v2.0.9.js

```diff
@@ -0,0 +1,229 @@
+            name: 'Bounty',
+            symbol: "~",
+        }
+    }
+
+    const ownerAddress = argv.ownerAddress || process.env.DEPLOY_OWNER_ADDRESS || await web3Connection.getAddress();
+
+    const getNetworkReceipt = async () =>
+        argv.networkToken || (await Deployer(ERC20, [tokens.network.name, tokens.network.symbol, tokens.network.cap, ownerAddress]))?.contractAddress;
+    const getTransactionalReceipt = async () =>
+        argv.transactionalAddress || (await Deployer(ERC20, [tokens.transactional.name, tokens.transactional.symbol, tokens.transactional.cap, ownerAddress]))?.contractAddress;
+
+    try {
+        // 1. Deploying Contracts
+        console.log(`Deploying Contracts...`);
+        const networkReceiptAddress = await getNetworkReceipt();
+        const transactionalReceiptAddress = await getTransactionalReceipt();
+        const rewardReceiptAddress = (await Deployer(ERC20, [tokens.reward.name, tokens.reward.symbol, tokens.reward.cap, ownerAddress]))?.contractAddress
+
+        const nftReceiptAddress = (await Deployer(BountyToken, [tokens.nft.name, tokens.nft.symbol]))?.contractAddress
+
+        // 2. Instancing Contract
+        console.log(`Loading Contracts...`);
+        const networkToken = new ERC20(web3Connection, networkReceiptAddress);
+        const bountyTransactional = new ERC20(web3Connection, transactionalReceiptAddress);
+        const rewardToken = new ERC20(web3Connection, rewardReceiptAddress);
+
+        const bountyToken = new BountyToken(web3Connection, nftReceiptAddress);
+
+        await networkToken.loadContract();
+        await bountyTransactional.loadContract();
+        await rewardToken.loadContract();
+        await bountyToken.loadContract();
+
+        // Transfer BEPRO to dev accounts
+        if (argv.network !== 'custom' || argv.network !== 'local')
+            for (const address of stagingAccounts) {
+                console.log(`Transfering 10M BEPRO to ${address}`);
+                await bountyTransactional.transferTokenAmount(address, 10000);
+            }
+
+
+        console.log(`Deploying Network_V2 With Registry...`);
+        const registryReceipt = await Deployer(NetworkRegistry, [networkToken.contractAddress, 1000, ownerAddress, 10000]);
+        const networkReceipt = await Deployer(Network_v2, [networkToken.contractAddress, registryReceipt.contractAddress]);
+
+        const network = new Network_v2(web3Connection, networkReceipt.contractAddress);
+        await network.loadContract();
+
+        console.log(`Changing Network_V2 Settings...`);
+        await network.changeDraftTime(60 * 5);//  5 minutes
+        await network.changeDisputableTime(60 * 10); // 10 minutes
+        await network.changeCouncilAmount(105000); // 105000 Tokens
+
+        //add allowed tokens
+        console.log(`Adding Allowed Tokens...`);
+        network.registry.addAllowedTokens([networkToken.contractAddress, rewardToken.contractAddress, bountyTransactional.contractAddress]);
```

Comment:
missing the same call but for `allowedTokens([], TRUE)` because what this statement does is add _allowed reward tokens_